src/praisonai-agents/praisonaiagents/session/sqlite_transcript_store.py

```
@@ -0,0 +1,472 @@
+            if self.db_path != ":memory:":
+                os.makedirs(os.path.dirname(self.db_path) or ".", exist_ok=True)
+            conn = sqlite3.connect(
+                self.db_path, check_same_thread=False, isolation_level=None
+            )
+            try:
+                # WAL lets readers proceed concurrently with a writer.
+                conn.execute("PRAGMA journal_mode=WAL")
+                conn.execute("PRAGMA synchronous=NORMAL")
+                conn.execute("PRAGMA busy_timeout=%d" % int(self.lock_timeout * 1000))
+            except Exception:  # pragma: no cover - PRAGMA best-effort
+                pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)